modules/ROOT/pages/index.adoc

```diff
@@ -1,81 +1,78 @@
 [[index]]
+:description: This section describes the Neo4j GraphQL Library.
 = Neo4j GraphQL Library
-:experimental:
-:chapter-label:
-:toc-title: Contents
-//:front-cover-image: image::title-page.png[]
-:header-title: NEO4J GRAPHQL LIBRARY
-:title-page-background-image: image::title-page.png[]
 
 The Neo4j GraphQL Library is a highly flexible, low-code, open source JavaScript library that enables rapid API development for cross-platform and mobile applications by tapping into the power of connected data.
 
-With Neo4j as the graph database, the GraphQL Library makes it simple for applications to have application data treated as a graph natively from the front-end all the way to storage, avoiding duplicate schema work and ensuring flawless integration between front-end and backend developers.
+With Neo4j as the graph database, the GraphQL Library makes it simple for applications to have data treated as a graph natively from the front-end all the way to storage.
+This avoids duplicate schema work and ensures flawless integration between front-end and backend developers.
```

Comment:
Super picky but, do you think we should be consistent with hyphenation? front-end and back-end, or frontend and backend?

Reply:
No, it's a good catch. I'm gonna check with the rest of the team which one we are using officially. Thanks for flagging it!